opentracing-noop/src/main/java/io/opentracing/noop/NoopTracer.java

```diff
@@ -44,7 +55,7 @@ public ActiveSpan activeSpan() {
 
     @Override
     public ActiveSpan makeActive(Span span) {
```

Comment:
For the observer usecase it is not necessary - tried returning `NoopActiveSpan.INSTANCE` but it caused problems with the mock tracer tests.